### DISCUSSION TITLE
과제 2번 관련 문의드립니다

### DISCUSSION THREAD

**Discussion by HanyeongYu** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#133] [2024년도 기타 질문]
12번에서 데이터라벨링을 한 것을 바탕으로 과제 2번에서 캡쳐하여서 제출하라고 하였는데 다운받은 데이터셋의 양이 900개가 넘는데 이 데이터셋을 전부 라벨링한 뒤에 캡쳐하여서 제출하는지 아니면 일부만 라벨링하여서 캡쳐해도 되는지 문의드립니다.

**Comment by HyeongKeunHong** [comment]
일부만 하셔서 제출해도 무방합니다. 다만, 이후 과제를 위해 충분한 양의 데이터에 라벨링은 하셔야 합니다.

**Reply by HanyeongYu** [reply]
네 알겠습니다!
